src/game_objects/game_object.cpp

```diff
@@ -28,8 +63,29 @@ void GameObject::connect_skel_to_model() {
     model->set_bones(&skel);
 }
 
+
+// Sets the material color for the entire game object's model
 void GameObject::set_color(Color color) {
     for (Mesh* mesh : model->meshes) {
         mesh->material->diffuse = color;
     }
-}
\ No newline at end of file
+}
+
+// Sets position of both object's transform and rigid body
+// Use this intead of transform.set_position to make sure that rigid body
+// is located in the same place as the object
+void GameObject::set_position(glm::vec3 pos) {
```

Comment:
Alternative is to have GameObjects instead have a RigidTransform thing that has both behavior of Transform and knows about a Rigidbody. But it's okay.